test-app/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM node:lts-iron as builder
+
+LABEL maintainer="info@redpencil.io"
+
+# Do npm installs first to allow for caching
+WORKDIR /app
+COPY package.json package-lock.json ./
+COPY embeddable-say-editor/package.json embeddable-say-editor/package-lock.json embeddable-say-editor/
+COPY test-app/package.json test-app/
+RUN npm ci
+COPY . .
+
+# Pack module
```

Comment:
I'm wondering if it is necessary to `pack` the module seperately? Couldn't we just run `npm run build` at the monorepo root (which should link the `embeddable-say-editor` module automatically).

Reply:
We could e.g. replace it by the following:
```
RUN npm run build --workspace embeddable-say-editor
RUN npm run build --workspace test-app
```

Reply:
I wanted to make sure that, as much as possible, we are following the real npm installed process, to catch, for example, if a file is missing from the packed module.

Reply:
Yeah ok that makes sense!